<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChefSignUpDialog</class>
 <widget class="QDialog" name="ChefSignUpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1272</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="surnameLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>480</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="birthDateEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>530</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>590</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="experienceYearsLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>650</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="experienceLinkLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>710</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="signupButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>490</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>SignUp</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
